desktop/components/split_test/running_tests.coffee

```diff
@@ -14,4 +14,9 @@
 # this should export empty Object
 # module.exports = {}
 
-module.exports = {}
+module.exports =
```

Comment:
You may want to also do `edge: 'highlighted'` to force admins to see the new bar